views/layouts/main.hbs

```diff
@@ -159,7 +176,7 @@
 							window.location.href = result.url ;
 						}
 						else {
-							$("body").html('<div class="ui middle aligned center aligned grid"><div class="column"><div class="ui centered green card"><div class="content"><h1 class="header" style="color: #229A33 !important; font-size: 175%">Success</h1></div><p class="ui content text" style="font-size: 120%">'+result.url+'</p></div></div></div>')
+							$("body").html('<div class="ui middle aligned center aligned grid"><div class="column"><div class="ui centered green card"><div class="content"><h1 class="header" style="color: #229A33 !important; font-size: 175%">Success</h1></div><p class="ui content text" style="font-size: 120%">'+result.url+'</p><p class="ui content text" style="font-size: 120%"><button onClick="window.location.reload(true)" type="button">Go Back</button></p></div></div></div>')
```

Comment:
Potential XSS - use jquery templates or dynamically create elements and use $.text(result.url)